### PR TITLE
feat: [DBOPS-592]: LiquibaseCommandStep

### PR DESCRIPTION
**Pull request by hemanthbsridhar**
LiquibaseCommand step static yaml schema
``` yaml
- step:
    name: Run Liquibase Command
    type: LiquibaseCommandStep
    spec:
      schema: myschema
      instance: myinstance
      command: <..> ( eg. tag / update-one-sql )
      settings:
            setting_name: setting_value
```

### CODE REVIEW

File: v0/pipeline/steps/common/dbops-liquibase-command-step-info.yaml [hunk 1]
```diff
@@ -0,0 +1,124 @@
+title: LiquibaseCommandStepInfo
+allOf:
+- $ref: ../../common/step-spec-type.yaml
+- type: object
+  required:
+  - connectorRef
+  - dbSchema
+  - dbInstance
+  - command
+  - settings
```
Comment (archit-harness) [inline v0/pipeline/steps/common/dbops-liquibase-command-step-info.yaml:10]: why is settings required?

File: v1/pipeline/steps/common/dbops-liquibase-command-step-info.yaml [hunk 2]
```diff
@@ -0,0 +1,124 @@
+title: LiquibaseCommandStepInfo
```
Comment (archit-harness) [inline v1/pipeline/steps/common/dbops-liquibase-command-step-info.yaml:1]: v1 not required

File: v0/pipeline/stages/custom/execution-wrapper-config.yaml [hunk 3]
```diff
@@ -76,6 +76,7 @@ properties:
     - $ref: ../../steps/common/aws-cdk-rollback-step-node.yaml
     - $ref: ../../steps/cd/download-aws-s3-step-node.yaml
     - $ref: ../../steps/cd/download-harness-store-step-node.yaml
+    - $ref: ../../steps/common/dbops-liquibase-command-step-node.yaml
```
Comment (Abhishek-CDC) [inline v0/pipeline/stages/custom/execution-wrapper-config.yaml:79]: why we are adding only to custom?
I think apply and rollback are added to cd and approval stages also.

Comment (hemanthbsridhar) [reply v0/pipeline/stages/custom/execution-wrapper-config.yaml:79]: updated the step node to both cd and approval stages.

File: v1/pipeline/steps/common/dbops-liquibase-command-step-info.yaml (hunk 2 above)
Comment (hemanthbsridhar) [reply v1/pipeline/steps/common/dbops-liquibase-command-step-info.yaml]: updated. v1 changes are reverted.

**Review by archit-harness — APPROVED** [review]

**Pull request merged**